### ISSUE TITLE
Remote: Rez Inventory non functional

### ISSUE THREAD

**Issue by DogThingTypePerson** [OpenCollarTeam/OpenCollar#617]
What version of OpenCollar are you using? 8.1

What behavior did you expect? To give a list of contents to rez

What behavior did you see instead? Nada, Zilch, Nuffin'

What steps does someone need to take to reproduce the problem? Add a leash post or whatever, full perms, to the Remote's contents. Press button. Voila!

**Comment by Trinkitz** [comment]
I tried this in 3 different mainland rez areas, and I was also unable to get a full-perm object to rez from the remote's contents.

**Comment by tayaphidoux** [comment]
wonder if the issue is the same one i came across when building my addon unit that rezes items for me, if you are checking for group permisisons it can cause issues i had to coment out that check form code i found to check if you can rez  before doing so, below is the function that does check i think i got it from lslwiki when reserachign rezing, note i comented out the check for same group as parcel group.


```
integer g_iCanRezAt(vector pos)
{
    integer parcelFlags = llGetParcelFlags(pos);
    list parcelDetails = llGetParcelDetails(pos, [PARCEL_DETAILS_OWNER, PARCEL_DETAILS_GROUP]);
    key parcelOwner = llList2Key(parcelDetails, 0);
    //key parcelGroup = llList2Key(parcelDetails, 1);

    if (parcelFlags & PARCEL_FLAG_ALLOW_CREATE_OBJECTS) return TRUE;
    if (parcelOwner == llGetOwner()) return TRUE;
    //if (!llSameGroup(parcelGroup)) return FALSE;
    if (parcelFlags & PARCEL_FLAG_ALLOW_CREATE_GROUP_OBJECTS) return TRUE;
    return FALSE;
}
```

**Comment by Mistress-Dalgato** [comment]
did this issue ever get fix? this problem is occurring in 8.3. box button on remote that should bring a menu to rez leash post does not work

**Comment by Mistress-Dalgato** [comment]
it looks like no rez button or rez command was put back in the code, perhaps taken out.

**Comment by Medea-Destiny** [comment]
@Mistress-Dalgato Yes, this function seems to have vanished from the code since version 8.0. I think the thinking was that because you can leash a collar wearer to anything, the main purpose of it (rezzing a leash post) seemed superfluous and potentially confusing (because you obviously need to have rez permissions for it to work). 

People have very rarely asked about it, so I suspect it's not a particularly in-demand feature, though it's odd that we retain a button for it in the remote. Perhaps we could re-introduce it as an optional app?

**Comment by Mistress-Dalgato** [comment]
@Medea-Destiny I'm so used to the function because I used it alot. Now the remote has a dead button that does nothing like a few other buttons.

I had a script friend attempt to add in a line in a code to where you click leash, click rez post and it was suppose to rez a post. Sadly we couldn't get it to rez the post.

An app could be fun as long as it works with the remote. Because in 7.0.1 you could rez anything including a leash holder.

**Comment by Medea-Destiny** [comment]
I have provided @Mistress-Dalgato a basic fix for this, will look at expanding it a little to handle nay object (right now it's just rezzing a post) and putting it back in.

**Comment by Mistress-Dalgato** [comment]
> I have provided [@Mistress-Dalgato](https://github.com/Mistress-Dalgato) a basic fix for this, will look at expanding it a little to handle nay object (right now it's just rezzing a post) and putting it back in.


thank you for the script again, but i didnt know it could take it back to the remote lol. I just thought rez it out, delete when done lol or keep it out. thank you nonetheless.

**Comment by Medea-Destiny** [comment]
@Mistress-Dalgato Oh I meant put the function back into the remote, not have the remote pick the object up again!  :)

**Comment by Mistress-Dalgato** [comment]
> [@Mistress-Dalgato](https://github.com/Mistress-Dalgato) Oh I meant put the function back into the remote, not have the remote pick the object up again! :)

that clears it up lol, thank you.
